src/txmempool.h

```diff
@@ -63,8 +77,16 @@ struct LockPoints
 
 class CTxMemPoolEntry
 {
+public:
+    typedef std::reference_wrapper<const CTxMemPoolEntry> CTxMemPoolEntryRef;
+    // two aliases, should the types ever diverge
+    typedef std::set<CTxMemPoolEntryRef, CompareIteratorByHash> Parents;
+    typedef std::set<CTxMemPoolEntryRef, CompareIteratorByHash> Children;
```

Comment:
nit: Can use C++11 :sweat_smile: 

```suggestion
    using Children = std::set<CTxMemPoolEntryRef, CompareIteratorByHash>;
```

Reply:
Good review comments above. I'll point out, though, that git grepping "typedef" seems much better for git grepping the codebase than "using" (yields more useful results).

Reply:
`git grep` can also do regex, so a `git grep typedef` would be `git grep --extended-regexp  'using \w+ ='`, and we are already using `using` quite extensively ;)

Reply:
i don't have any preference.